src/presentations/indexable-presentation.php

```diff
@@ -223,14 +224,23 @@ public function set_helpers(
 	 *
 	 * @return string The permalink.
 	 */
-	public function get_permalink() {
+	public function generate_permalink() {
 		if ( $this->indexable_helper->dynamic_permalinks_enabled() ) {
 			return $this->permalink_helper->get_permalink_for_indexable( $this->model );
 		}
 
 		return $this->model->permalink;
 	}
 
+	/**
+	 * Helper function to retrieve the permalink.
+	 *
+	 * @return string
+	 */
+	public function get_permalink() {
```

Comment:
We should deprecate this function as it's now identical to `->permalink`.

Reply:
Done